spec/models/review_spec.rb

```diff
@@ -0,0 +1,30 @@
+require 'rails_helper'
+
+RSpec.describe Review, type: :model do
+  before :each do
+    @good_review = Review.create(product_id: 1, rating: 5)
+    @bad_review = Review.create(product_id: nil, rating: nil)
+    @negative_nums = Review.create(product_id: 1, rating: -1)
+    @high_nums = Review.create(product_id: 1, rating: 10)
+  end
+
+  describe "validations" do 
+    it "good is good and bad is bad" do 
```

Comment:
I think it would make more sense to have these two be two separate tests since the fact that one is good and the other is bad is not necessarily directly related to one another.